external/mmdetection/detection_tasks/apis/detection/nncf_task.py

```diff
@@ -230,12 +230,6 @@ def optimize(
         self._is_training = True
         self._model.train()
 
-        fp16 = training_config.get("fp16", None)
```

Comment:
Should NNCF version be updated for this changes?

Reply:
Yes, it should

Reply:
Let's update it here